### PR TITLE
log killed queries

### PR DESCRIPTION
**Pull request by bogensberger**

### CODE REVIEW

File: sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java [hunk 1]
```diff
@@ -1808,6 +1809,55 @@ public void testNewColumn() throws Exception {
...
     }
+
+    @Test
+    public void testKilledJobLog() throws Exception {
+        execute("create table likes (" +
+                "   event_id string," +
+                "   item_id string" +
+                ") clustered into 1 shards with (number_of_replicas = 0)");
+        ensureYellow();
+        execute("SET GLOBAL stats.enabled = true");
```
Comment (mfussenegger) [inline sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java]: stats should be disabled in a safe way (if an exception/assertion error occurs) after the test run

File: sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java [hunk 2]
```diff
@@ -1808,6 +1809,55 @@ public void testNewColumn() throws Exception {
...
+        final String stmt = "insert into likes (event_id, item_id) values ('event1', 'item1')";
+
+        Thread insertThread = new Thread(new Runnable() {
+            @Override
+            public void run() {
+                for (int i=0; i < 100; i++) {
+                    try {
+                        execute(stmt);
+                    } catch (SQLActionException e) {
+                        e.getCause();
```
Comment (mfussenegger) [inline sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java]: `e.getCause()` ?

File: sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java (hunk 1 above)
Comment (mfelsche) [inline sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java]: move to separate test and use SUITE or TEST cluster scope

**Comment by mfelsche** [comment]
self-ack

**Pull request merged**